.circleci/scripts/check_faucet_balance_and_warn.py

```diff
@@ -0,0 +1,63 @@
+import json
+import os
+from urllib import request, parse
+
+THRESHOLD = 3000000000000000000 # 3 ETH
+FAUCET_ADDRESS = '0x179E20BA056FF73A8161d3E3bbC1627CB12e88cC'
```

Comment:
this should probably be configurable?

Reply:
hmmm so you are proposing this to be as env var and set in CircleCI config instead?

Reply:
yeah, probably

Reply:
late to come back but updated